### DISCUSSION TITLE
How to disable automatic styles and set my own?

### DISCUSSION THREAD

**Discussion by Kanape21** [lllyasviel/Fooocus#3377] [Q&A] [answered]
The translation is done by deepl, sorry.
How to remove the inclusion of styles when using this feature?
![image](https://github.com/user-attachments/assets/215e62b8-3ae3-4de0-9e3b-d5bb45727cdd)
I don't like the promts that are recorded in those styles. 
Is there any way to make my own?
![image](https://github.com/user-attachments/assets/cfb9bd21-d547-4b53-a438-aa46863f8889)
It's awfully inconvenient to have to go into the styles tab and turn it off all the time.

**Comment by mashb1t** [comment, marked as answer]
You can adjust the code in https://github.com/lllyasviel/Fooocus/blob/main/webui.py#L1064-L1071
